website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -4,72 +4,57 @@ page_title: Notifications - Workspaces - Terraform Cloud and Terraform Enterpris
 
 # Run Notifications
 
-Terraform Cloud can use webhooks to notify external systems about the progress of runs.
+Terraform Cloud can use webhooks to notify external systems about the progress of runs. Each workspace has its own notification settings, and can notify up to 20 destinations.
```

Comment:
Small nitpick: we should probably adjust the wording of the title and overview here to include workspace events since notifications are no longer a run-specific thing!

Reply:
ohh yeah!! that's a good point!! let me do that